grails-app/views/productAssociation/create.gsp

```diff
@@ -75,6 +75,15 @@
 									<g:textArea class="text" name="comments" value="${productAssociationInstance?.comments}" />
 								</td>
 							</tr>
+
+							<tr class="prop">
+								<td class="name">
+									<label><warehouse:message code="productAssociation.mutualAssociation.label" default="Two-way Association"/></label>
```

Comment:
"Two-way association" needs some work. Bi-directional is the only thing I can think of and that's not any better.

Reply:
Ok sounds good. My vote would be for Bi-directional association or Symmetric association. Or even something more laymen like "Do you want to create an inverse relationship for the associated product?"